frontend/e2e/pages/metrics/report-step.ts

```diff
@@ -456,4 +609,40 @@ export class ReportStep {
     await this.homeIcon.click();
     await expect(this.page).toHaveURL(/\//);
   }
+
+  async checkDisplayChartStatus() {
+    await this.displayListTab.click();
+
+    expect(await this.displayListTab.getAttribute('aria-selected')).toEqual('true');
+    expect(await this.displayChartTab.getAttribute('aria-selected')).toEqual('false');
+    await expect(this.chartTabsContainer).not.toBeVisible();
+
+    await this.displayChartTab.click();
+
+    expect(await this.displayListTab.getAttribute('aria-selected')).toEqual('false');
+    expect(await this.displayChartTab.getAttribute('aria-selected')).toEqual('true');
+    await expect(this.chartTabsContainer).toBeVisible();
+    expect(await this.displayBoardChartTab.getAttribute('aria-selected')).toEqual('true');
+    expect(await this.displayDoraChartTab.getAttribute('aria-selected')).toEqual('false');
+    await expect(this.velocityChart).toBeVisible();
+    await expect(this.averageCycleTimeChart).toBeVisible();
+    await expect(this.cycleTimeAllocationChart).toBeVisible();
+    await expect(this.reworkChart).toBeVisible();
+
+    await this.displayDoraChartTab.click();
+
+    expect(await this.displayBoardChartTab.getAttribute('aria-selected')).toEqual('false');
+    expect(await this.displayDoraChartTab.getAttribute('aria-selected')).toEqual('true');
+
+    await this.displayListTab.click();
+    await this.displayChartTab.click();
+
+    expect(await this.displayBoardChartTab.getAttribute('aria-selected')).toEqual('false');
+    expect(await this.displayDoraChartTab.getAttribute('aria-selected')).toEqual('true');
+
+    await expect(this.leadTimeForChangeChart).toBeVisible();
+    await expect(this.deploymentFrequencyChart).toBeVisible();
+    await expect(this.changeFailureRateChart).toBeVisible();
+    await expect(this.meanTimeToRecoveryChart).toBeVisible();
```

Comment:
split it into diff parts.
1. actions like `await this.displayListTab.click();` should not in `check.....`
2. assertion part should be in a function named like `check/validate.....`

Reply:
and the flow of operation should be ordered by the e2e flow, not in this page(report-step)